### PR TITLE
Orientation image verification

### LINKED ISSUE #8: FLS image flipped/mirrored when viewing through rqt image view
The issue can be seen on the gif in the Readme.md file. The cylinder is indicated that it's on the right of the sphere when it should be the other way round.

### PR DESCRIPTION
**Pull request by ivan1993br**
Correction of the flipped sonar image

close #8

### CODE REVIEW

File: src/FLSonar.cc [hunk 1]
```diff
@@ -569,7 +570,8 @@ void FLSonar::GenerateTransferTable(std::vector<int> &_transfer)
       else
       {
         this->sonarImageMask.at<uchar>(j, i) = 255;
-        int idBeam = static_cast<int>(((theta + M_PI) / (2 * M_PI)) * (this->beamCount - 1));
+        int idBeam = static_cast<int>(((theta + this->vfov / 2) / (this->vfov)) * (this->beamCount));
+        // int idBeam = static_cast<int>(((theta + M_PI) / (2 * M_PI)) * (this->beamCount - 1));
```
[the remark this answers is not in the document]
Comment (ivan1993br) [reply src/FLSonar.cc]: the commented line?

[the remark this answers is not in the document]
Comment (g-arjones) [reply src/FLSonar.cc]: Yes...

[the remark this answers is not in the document]
Comment (ivan1993br) [reply src/FLSonar.cc]: Fixed

**Comment by g-arjones** [comment]
Looks good.. :+1:

**Pull request merged**